PIM_BE/PIMTool.Core/Interfaces/Services/IProjectService.cs

```diff
@@ -0,0 +1,22 @@
+﻿using PIMTool.Core.Domain.Entities;
+using PIMTool.Core.Domain.Objects;
+
+namespace PIMTool.Core.Interfaces.Services
+{
+    public interface IProjectService
+    {
+        Task<Project?> GetAsync(int id);
+        Task<IEnumerable<Project>> GetAllAsync();
+        Task Create(Project project);
+        Task Update(Project project);
+        Task Delete(int id);
+        //Task<IEnumerable<Project>> SearchProject(string? searchText, string searchStatus, string sortNumber, string sortName, string sortStatus, string sortCustomer, string sortStartDate);
+        PagingDto SearchProjectV2(int pageSize, int pageIndex, string? searchText, string searchStatus, string sortNumber, string sortName, string sortStatus, string sortCustomer, string sortStartDate);
```

Comment:
[S] it is bad with long parameter like that.
It cannot support for general case when user want to filter more and sort more